modules/ROOT/pages/bare-metal.adoc

```diff
@@ -113,3 +117,73 @@ IPAPPEND 2
 == PXE rootfs image
 
 include::pxe-artifacts.adoc[]
+
+== Automated ISO/PXE installs with Ignition embedding
+
+The Fedora CoreOS Live environment is also CoreOS in the sense that it can boot via Ignition, execute containers, etc.  It includes all of the same content.
+
+As noted above for Live PXE, the live environment (including ISO) does not have to actually perform 
+a persistent installation.  You can boot it from a read-only medium (physical ISO, USB stick with a physical read-only switch),
+and do everything you do on any other Fedora CoreOS environment.
+Each boot will re-run the Ignition config.
+
+For the ISO, the mechanism to do this is `coreos-installer ignition iso embed`, which will create a new `.iso` file that combines your configuration with the ISO.
+Similarly, there is `coreos-installer pxe ignition wrap` for the PXE case.
+
+However, many system administrators will want to perform fully unattended persistent installations instead of running stateless.
+
+Some documentation on this is on the upstream installer site: https://coreos.github.io/coreos-installer/customizing-install/
+
+To emphasize, there are *two* Ignition configurations here; the first config ("ISO Ignition") will commonly embed a second rendered configuration that runs on the "target" installation.
```

Comment:
I usually call these the "live Ignition config" and the "installed Ignition config" ("target" also wfm).